runtime/tests/jvmtitests/agent/version.c

```diff
@@ -60,10 +62,20 @@ getVersionName(agentEnv * agent_env, jint version)
 			return versionNames[1];
 		case JVMTI_VERSION_1_2:
 			return versionNames[2];
-		case JVMTI_VERSION_9:
+		case JVMTI_1_2_SPEC_VERSION:
```

Comment:
The switch uses value `version & ~JVMTI_VERSION_MASK_MICRO` so this case will never occur.

Reply:
Agreed, this is same as the condition value within `jvmtiHookGetEnv()`, removed `JVMTI_1_2_SPEC_VERSION`.